agreement/views/agreement_type.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <!-- Agreement Type List View-->
+    <record id="agreement_type_list_view" model="ir.ui.view">
+        <field name="name">Agreement Type List</field>
+        <field name="model">agreement.type</field>
+        <field name="arch" type="xml">
+            <tree string="Agreement Types" default_order="name">
+                <field name="name" string="Type Name" />
+                <field name="domain" />
+            </tree>
+        </field>
+    </record>
+
+    <!-- Agreement Type Form View -->
+    <record id="agreement_type_form_view" model="ir.ui.view">
+        <field name="name">Agreement Type Form</field>
+        <field name="model">agreement.type</field>
+        <field name="arch" type="xml">
+            <form string="Agreement Type">
+                <sheet>
```

Comment:
Please add 
`<div class="oe_button_box" name="button_box"/>` so other modules can add smartbuttons to main view

Reply:
done